CHANGELOG.md

```diff
@@ -6,14 +6,20 @@ adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+
```

Comment:
I appreciate the extra whitespace in here 👍

Reply:
I hate to be the one to nitpick, but the old whitespace actually conforms to the [Keep a Changelog spec](https://keepachangelog.com/en/1.0.0/), which is why there wasn't any. We use the same format across all of our projects, so changing it for just this repos doesn't feel right to me. Also, it doesn't feel right to change it as a result of an unrelated PR. Can we remove the whitespace? Thanks!